### ISSUE TITLE
Is the GUI only for Windows?

### ISSUE THREAD

**Issue by raabuchanan** [abel-research/ampscan#78]
I'm trying to run the GUI in linux and the program just hangs. I need to kill the process externally to stop it.

I successfully installed using pip and then run 
`python ampscanGUI.py`

and nothing happens and I can't kill the program.

**Comment by JoshuaSteer** [comment]
I am not sure if we've ever tested the gui on linux. Can you get the ampscan functions working without the GUI?

**Comment by raabuchanan** [comment]
Yes, I can import the library and run the example python code. It's just ampscanGUI.py that doesn't seem to work.

**Comment by JoshuaSteer** [comment]
Have you run python-based GUIs from your linux build before? I think I ran it on Ubuntu years ago and cannot think of any particular reason why it would not work, but my knowledge in this space is limited.
